docs/plugins/export.rst

```diff
@@ -63,3 +73,15 @@ The default options look like this::
                 indent: 4
                 separators: [',' , ': ']
                 sort_keys: true
+        csv:
+            formatting:
+                ensure_ascii: False
+                indent: 0
+                separators: [',']
+                sort_keys: true
+        xml:
+            formatting:
+                ensure_ascii: False
+                indent: 4
+                separators: ['>']
+                sort_keys: true
```

Comment:
As above, I don't think these options are actually used for the new formats.

Reply:
I updated them to work correctly now with the added formats CSV and XML